ecpp

```diff
@@ -1,35 +1,75 @@
 #!/usr/bin/python
 
+# Author: Hubert Kario, Stefan Dordevic
+# Released under Gnu GPL v2.1, see LICENSE file for details
+
 from __future__ import print_function
 import ConfigParser
 import sys
 import StringIO
 from verify import *
+from convert import *
 import getopt
+import os
+import re
 
 # xrange python3 compatibility
 try:
     xrange
 except NameError:
     xrange = range
 
+# input files directory for primo tool
+PRIMO_IN = 'in/'
+CERTS_DIR = 'certificates/'
+
 # dict with params
-params = {}
+PARAMS = {}
```

Comment:
nit: top-level funcion and class definitions should have 2 blank lines before their definition